src/v2/controllers/FormController.js

```diff
@@ -48,17 +48,34 @@ export default Controller.extend({
   },
 
   invokeAction (actionPath = '') {
-    const actionFn = this.options.appState.getActionByPath(actionPath);
+    const idx = this.options.appState.get('idx');
+    if (idx['neededToProceed'][actionPath]) {
+      idx.proceed(actionPath, {}).then((resp) => {
```

Comment:
nit: 
```
idx.proceed(actionPath, {})
  .then(resp => {
```

Reply:
Why is there no data passed in?

Reply:
Previously we just called `actionFn ` now we are doing `idx.proceed(actionPath, {})`. This is slightly different than calling idx.proceed in the `handleFormSave ` since this is not a remediation action and one of the out of bound actions like `cancel` `factor-resend` `factor-poll` etc hence there is no data other than the stateHandle which idx-js knows to send.

Reply:
Interesting. So we are making assumption that only stateHandle is sent even though it can be a POST request? Maybe the IDX response can be improved, but even currently, in one of the device authenticator flow, we need to make POST request with challengeRequest data. We are not using the invokeAction to do so now, but it is because we need to use timeout option in the request (this should be fixable).

https://github.com/okta/okta-signin-widget/blob/master/playground/mocks/idp/idx/data/identify-with-device-probing-loopback.json#L60

Reply:
But never mind, the request is not one of those 'rel-form', so it is probably not following the convention.